lib/templates/template.tex

```diff
@@ -1,5 +1,8 @@
 \documentclass[$if(fontsize)$$fontsize$$else$12pt$endif$,$if(lang)$$lang$,$endif$$if(papersize)$$papersize$,$endif$$for(classoption)$$classoption$$sep$,$endfor$]{$documentclass$}
 
+\providecommand{\tightlist}{%
```

Comment:
In fact, is it not just that Pandoc is using an extra package? Does the new default template actually have this definition inline?